monitoring/local_monitoring/local_monitoring_demo.sh

```diff
@@ -141,6 +141,88 @@ function kruize_local() {
 	echo
 	echo "######################################################"
 	echo
+
+	echo
+  echo "######################################################"
+  echo "#     Delete previously imported metadata"
+  echo "######################################################"
+  echo
+  curl -X DELETE http://"${KRUIZE_URL}"/dsmetadata \
```

Comment:
@shreyabiradar07 - You can remove delete metadata. Deploy another app into a new namespace and do an import metadata

Reply:
Have retained delete `/dsmetadata` API call to showcase `DELETE` functionality and added additional import metadata `POST /dsmetadata`  calls to showcase multiple import metadata functionality by creating a new namespace and deploying TFB benchmark in the new namespace created

Updated demo script execution output - https://privatebin.corp.redhat.com/?efc4bba2780cb09d#5NrFVVVx3ksogT2WJaXjP1QtMMH1S2mZPQbrZu4b5EeL

Reply:
@shreyabiradar07 
I still see the older error message in the output, wasn't this fixed to just have the parameters that were passed to list_dsmetadata?
{"message":"Metadata for a given datasource - \" prometheus-1 \", cluster name - \" null \", namespace - \"null \" either does not exist or is not valid","httpcode":400,"documentationLink":"","status":"ERROR"}

Also, I don't see any cpu recommendations in the output. Can you check this.

Reply:
> I still see the older error message in the output, wasn't this fixed to just have the parameters that were passed to list_dsmetadata?
> {"message":"Metadata for a given datasource - " prometheus-1 ", cluster name - " null ", namespace - "null " either does not exist or is not valid","httpcode":400,"documentationLink":"","status":"ERROR"}

Error message is fixed as part of #[1200](https://github.com/kruize/autotune/pull/1200) which is yet to be merged 

> Also, I don't see any cpu recommendations in the output. Can you check this.

The above output shared is by running the demo script on `minikube`. Will run the script on `openshift` and share the updated output

Reply:
@chandrams Demo script output on `openshift` - https://privatebin.corp.redhat.com/?11f2190fd5439274#CC2qpvigvJPAF2F3HEnUpH6frAKDmBmrwTED9dRfhmPc

Reply:
@shreyabiradar07 - I think with minikube certain versions have issues, you can try with kind. Openshift output looks fine.